docs/EN/Configuration/xdrip.md

```diff
@@ -0,0 +1,238 @@
+# xDrip+ settings
+
+If not already set up then download [xdrip](https://github.com/NightscoutFoundation/xDrip)
+
+For G6 transmitters manufactured after fall/end of 2018 please make sure to use one of the [latest xDrip+ versions](https://github.com/NightscoutFoundation/xDrip/releases). Those transmitters have a new firmware and latest stable version of xDrip+ (2019/01/10) cannot deal with it.
```

Comment:
"make sure to use one of the [latest xDrip+ versions]" > make sure to use one of the [latest nightly built xDrip+ versions]

Reply:
done